app/routes.php

```diff
@@ -50,6 +51,8 @@
             ->add(PlainRecaptchaMiddleware::class)
         ;
 
+        $versionGroup->get('/password-reset-token/{base58secret:[A-Za-z0-9-]{22}}', GetPasswordResetToken::class);
```

Comment:
Let's camel case to keep things consistent:
```suggestion
        $versionGroup->get('/password-reset-token/{base58Secret:[A-Za-z0-9-]{22}}', GetPasswordResetToken::class);
```